app/views/content/how-to-apply-for-teacher-training/subject-knowledge-enhancement.md

```diff
@@ -20,7 +20,7 @@ youtube_video:
     id: mWuHMtyFrFU
     title: Subject knowledge enhancement explained
 navigation: 30.25
-navigation_title: Subject knowledge enhancement
+navigation_title: Subject knowledge enhancement (SKE)
 navigation_description: Find out how to improve your subject knowledge with a subject knowledge enhancement (SKE) course.
```

Comment:
```suggestion
navigation_description: Find out how to improve your subject knowledge with a subject knowledge enhancement course.
```

Should we take `(SKE)` out of the description now that it is in the nav card title?